hashicorp-services/tfm
# Boiler Template and Code structure

# Pull request


## Related Issue

tag the issue number with `#number` syntax;
__issue__ #https://hashicorp.atlassian.net/jira/software/projects/ISINFRA/boards/224?selectedIssue=ISINFRA-108

## Description

### Scope:

- Directory structure and packages created and modeled after best practices.
- a client package for client contexts for source and destination file committed for review
- outputs package for outputs of logs and tables
- A migrate package with a working copy command to copy teams for review as boiler plate code for other copy commands.

## Expectation

please check all the relevant components;

- [ ] Syntax review
- [ ] I don't understand pre-commit validation __HELP__ (You have read the ./contributors.md in the project)
- [ ] Code review && test
- [ ] Something else see How To

## How to'

- How to Test, 
```
go run main.go list teams
go run main.go list orgs 
go run main.go copy teams
```

- How to what ever else you expect a reviewer to do for review.
